### DISCUSSION TITLE
ToS

### DISCUSSION THREAD

**Discussion by HaveDex** [Tyrrrz/YoutubeDownloader#279] [General]
I do not condemn Russia, because I literally do not care about either country. Putin was the aggressor in this case, but I do not care, nor is Zelensky a saint either. Zelensky is no better than Putin, with how corrupt he is (shutting down news stations, jailing opposition). Now he's begging the West for weapons and free money, while his begging is also, inadvertently, driving up prices even more, because there's no such thing as free weapons or money. Our governments shipping this clown all this shit will all come from our own pockets. Have you seen the goddamn prices for even basic ingredients and gas?

I am still going to use this DLer and I literally don't care about the fictitious "terms of service". Not beyond the point of telling you about my stance on it anyway.

Try focusing more on making good software like this instead of going "I SUPPORT CURRENT THING" while jamming your mouth open 180 degrees, because virtue-signaling and or being brainless like this will get you nowhere. Making good software that people would also pay for will.

Also, I do not care that this will be inevitably deleted. The left and brain-dead people love censorship, just like Putin and Zelensky. Can't take criticism or want others to see how things really are. And yes, this is criticism. The criticism is that you shouldn't put your worthless "I SUPPORT CURRENT THING" propaganda in something entirely unrelated. Imagine complaining about propaganda and other crap, when your own country's gov't does the same shit. If you senselessly follow those actions here as well you're no better than Putin or the little boy Zelensky.
And before you call this "harassment" for calling people brain-dead and etc., you do the same in your tags and the description. Can't have it both ways, for any of these things.

**Comment by bgeery** [comment]
Typical selfish and entitled right-winger.  Scum.

**Reply by Sobieck** [reply]
Yes, defending your homeland from a genocidal fascist is the current thing. And people in the west are suffering inflation because we are sending weapons. You are very smart, have a firm grasp of economics, and are very moral.

**Comment by neon-sunset** [comment]
@HaveDex just so you know, Ukraine has a large community of developers and @Tyrrrz, like many other people here, is from Ukraine. Your claim on virtue signaling couldn't be farther from the truth, and while I understand that the US has an issue with people supporting "current thing"s simply to get social points, this is not the case for us.
Moreover, you can still make a choice and stop regurgitating russian propaganda that benefits no one but them.

@Sobieck Are you? Do you know that a lot of military support is providing what's in inventory, and the financials - the 40bil bill means the support is prolonged across many months/years. Not a single transfer. And also, it's a measuring error against US economy size, it cannot negatively influence inflation. It is a public safety concern for everyone to make sure russia stops existing in its current form. Once this happens, whatever new breakaway state sells oil and gas can do so in the form of reparations to other nations.

**Comment by foxanna** [comment]
> Try focusing more on making good software like this instead of going "I SUPPORT CURRENT THING" 

Oh really? You know, @HaveDex, me, Oleksii, and thousands of Ukrainian developers have it as their strongest wish: live in peace and be able to focus on doing things we love like creating good software, as you suggest here. However, we CANNOT while there is a WAR in OUR country and while missiles literally fly above our heads. And some of us now had to switch our keyboards to weapons, leave our homes and families, and protect the souverenity and freedom of our nation. We'd all be glad to have another "CURRENT THING" in our minds and lives. In this light, your advice is nothing more than a disrespectful mockery.

**Comment by slimshader** [comment]
Imbecile "virtual signalling" remarks aside, also typical: a**hole with no single repo lectures about "making good software" while disregarding ToS.

**Comment by barisere** [comment]
@HaveDex it would be better if you did not try to lecture a Ukrainian about virtue signalling and the cost of gas. I understand your frustration. Putin is blackmailing the world with food scarcity and energy crisis while committing heinous atrocities in Ukraine.
If the cost of stopping the genocidal maniac is me eating less for a while and paying more, I believe it's worth doing. The higher-order effects of this aggression will echo around the world as more despots become emboldened.
If anyone should give a lecture about virtue signalling, it is us, non-US citizens who watch with horror as your society thrashes the values we grew up admiring it for.
I probably do not have the right to condemn your opinion either, I'm not an American. My country isn't giving anything to help. I am thankful that the United States exists and has committed so much to help, although you feel the pain. It's not an easy time for anyone.
Thanks for your opinion, I understand.

**Comment by alexyakunin** [comment]
Dear Mr. Entitled,

> I am still going to use this DLer and I literally don't care about the fictitious "terms of service". Not beyond the point of telling you about my stance on it anyway.

It's called "stealing", you know. "Free" doesn't mean it's free unconditionally. And if you're violating the use terms while using the software, you are plainly stealing it.

> Try focusing more on making good software like this instead of going "I SUPPORT CURRENT THING" 

Totally agree with @bgeery and @slimshader here. It's crazy you think it's fine to even throw such an advice here - assuming you are a thief from the legal standpoint.

> Also, I do not care that this will be inevitably deleted. The left and brain-dead people love censorship, just like Putin and Zelensky.

@Tyrrrz has literally zero reasons to delete this post - on contrary, it makes sense to keep it. People need to know there are folks like you. Maybe you don't get it, but Y-1941 version of your statement would be, "Hey jew, I don't care about the genocide Hitler commits against your people. I don't care about Hitler too. I think I am entitled to use the result of your intellectual work for free w/o agreeing with any political part of use terms - because, well, I don't care about this!"

The moral is: check out where the Russian warship went, please.

**Comment by Lerroxy** [comment]
Well, even though the code has been updated by people from all over the world, the program's still technically his, so it's in his full right to to write anything he wants in its ToS, no hard feelings.

However, the program doesn't do anything fancy and GUI isn't special, so if you don't like seeing emotional politics at your doorstep - here's some other options:

https://github.com/ytdl-org/youtube-dl/ - python command line tool. The lightest and easiest, but requires the use of terminal/powershell
https://github.com/yt-dlp/yt-dlp - the actively updated fork of the tool above

If you care about having more user-friendly interface:
https://github.com/MrS0m30n3/youtube-dl-gui - win/linux
https://clipgrab.org/ - mac (not sure about it being open source or not tho)

**Comment by LZeugirdor** [comment]
It's all fun and games until you find out that this software has a trojan program that DDOS' russian servers. I don't care if you support Ukraine, when you put programs like that into software like this you just can't trust it. It could be a DDOS or them sending info to Russia, I don't know and I don't care, that sort of thing has no place on github.

**Reply by bgeery** [reply]
OK, Russian troll.

**Reply by LZeugirdor** [reply]
https://www.virustotal.com/gui/file/817fb647f82a667b54f4554a677930996c24b03aad003fed7962ccd0a7997837/relations
https://www.virustotal.com/gui/ip-address/23.216.147.76/details
if you're that ready to back someone at least do your research.

**Reply by bgeery** [reply]
Cool.  Now I'll run a copy on each of my PCs.

**Reply by LZeugirdor** [reply]
If it does end up being Russian malware I wish you luck lmao.

**Reply by bgeery** [reply]
It's called false positives.  New to computers, huh?  Good luck, troll.

**Reply by LZeugirdor** [reply]
False positive for contacting a flagged IP address? You really are shilling for this app. My point is proven, have a good day.
> It's called false positives. New to computers, huh? Good luck, troll.
